portfolio/src/main/webapp/index.html

```diff
@@ -1,35 +1,49 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Personal Portfolio | Paul Colța</title>
     <link rel="stylesheet" href="style.css">
-
-    <title>Personal portfolio - Paul Colta!</title>
+    <script src="script.js"></script>
   </head>
-  <body>
-    <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
-  <ol class="carousel-indicators">
-    <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
-    <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
-    <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
-  </ol>
-  <div class="carousel-inner">
-    <div class="carousel-item active">
-      <img class="d-block w-100" src="img/picture_of_me.jpg" alt="First slide">
-      <div class="carousel-caption">
-        <h5 class="alert alert-light">Hi, I'm Paul and this is my portfolio. 😄</h5>
-        <div class="col-md-4 text-center"> 
-            <label for="commessages ments-number">Number of messages</label>
-            <input type="number" id="messages-number" name="messages-number" min="1" max="100" required>
-            <button onclick="getMessagesFromServer()" class="btn btn-primary">Get Messages</button>
-            <ul class="list-group" id="messages-container"></ul>
+  <body style="background-color: #435969;">
+    <div id="content">
+     <div id= "header">
+      <img src="img/picture_of_me.jpg" alt="Paul" style="width:300px;height:180px;">  
+      <h1></h1>
+      <p>Hi! I'm a <b>STEP Intern</b> in the Plx ZRH team. I'm from Romania, a country from the Southestern part of Europe. </br>
+      <div id="buttonline">
+        <button onclick="showAbout()">About Me</button>
+        <button onclick="showSkills()"> Skills </button>
```

Comment:
``<button onclick="showSkills()"> Skills </button>``

Remove the spaces between ``> Skills <``, it should be

``<button onclick="showSkills()">Skills</button>``

Some for the other tags in this file.

Reply:
Solved